cns/ipampool/metrics.go

```diff
@@ -133,4 +144,9 @@ func observeIPPoolState(state ipPoolState, meta metaState) {
 	ipamPrimaryIPCount.WithLabelValues(labels...).Set(float64(len(meta.primaryIPAddresses)))
 	ipamRequestedIPConfigCount.WithLabelValues(labels...).Set(float64(state.requestedIPs))
 	ipamTotalIPCount.WithLabelValues(labels...).Set(float64(state.totalIPs))
+	if meta.exhausted {
+		ipamSubnetExhaustionState.WithLabelValues(labels...).Set(float64(subnetIPExhausted))
```

Comment:
Do we need two metrics? This is a gauge then counter metric, so cant we determine subnet exhaustion based on if the value is 1 or 0? @rbtr  Thoughts?